cosmic-settings/src/pages/time/date.rs

```diff
@@ -95,10 +145,30 @@ fn format() -> Section<crate::pages::Message> {
                         .toggler(page.military_time, Message::MilitaryTime),
                 )
                 // First day of week
-                .add(settings::item(
-                    &*section.descriptions[1],
-                    horizontal_space(Length::Fill),
-                ))
+                .add(
+                    settings::item::builder(&*section.descriptions[1]).control(dropdown(
```

Comment:
I'm trying to get away from referencing strings from the descriptions list. See other pages that are using static global strings.

Reply:
fixed in https://github.com/pop-os/cosmic-settings/pull/212/commits/3def7aeeba09440891bc199f019816559630b6f0